schema/me/conversation/message.js

```diff
@@ -84,12 +89,26 @@ export const MessageType = new GraphQLObjectType({
       type: new GraphQLList(AttachmentType),
     },
 
+    invoice: {
+      type: InvoiceType,
+      resolve: ({ metadata, conversation_id }, options, request, { rootValue: { conversationInvoiceLoader } }) => {
+        if (!isInvoiceMessage(metadata)) {
+          return null
+        }
+        return conversationInvoiceLoader({
+          conversation_id,
+          lewitt_invoice_id: metadata.lewitt_invoice_id,
+        }).then(invoice => {
+          return invoice
+        })
+      },
+    },
+
     is_invoice: {
       description: "True if message is an invoice message",
+      deprecationReason: "Prefer to use invoice.",
```

Comment:
Yea, I was thinking you'd want to render something differently with an invoice, so you'd naturally need ` invoice { state } ` and then `if invoice` for example, but yea- we shouldn't _require_ that, `is_invoice` as a stand-alone bool doesn't need to be deprecated. Will undo.